conf/application-json-logger.xml

```diff
@@ -5,9 +5,4 @@
         <encoder class="uk.gov.hmrc.play.logging.JsonEncoder"/>
     </appender>
 
-    <logger name="uk.gov" level="${logger.uk.gov:-WARN}"/>
```

Comment:
Think you keep these sections, but the pr commenter was talking about the use of ${} to look up properties.

Reply:
@ardwalker Modifed, thanks